app/assets/javascripts/media_player_wrapper/mejs4_helper_markers.es6

```diff
@@ -92,11 +98,12 @@ class MEJSMarkersHelper {
 
       // Show popover confirmation
       $button.popover({
-        container: '#popover-container-' + $button[0].dataset.markerId,
+        container: '#popover-container-' + markerId,
         content: content,
         sanitize: false,
         html: true,
-        placement: 'top'
+        placement: 'top',
+        trigger: 'focus',
```

Comment:
I think eslint is complaining about this trailing comma (https://codeclimate.com/github/avalonmediasystem/avalon/pull/4980)
```suggestion
        trigger: 'focus'
```